pypeit/telescopes.py

```diff
@@ -80,6 +80,14 @@ def __init__(self):
 # VLT aperture from https://www.eso.org/observing/etc/doc/formulabook/node15.html
 # This seems unrealistic given that pi(8.2^2)/4 = 52.81
 
+class NTTTelescopePar(TelescopePar):
+    def __init__(self):
+        super(NTTTelescopePar, self).__init__(name='NTT',
+                                              longitude=289.2700,
+                                              latitude=-29.2567,
+                                              elevation=2375,
+                                              diameter=3.58)
```

Comment:
Please see if you can find the effective aperture eff_aperture somewhere in the literature or in a manual. That is better than diameter for throughput calculations.

Reply:
Seems not related information in the literature...